TestAssets/TestPackages/dotnet-desktop-and-portable/dotnet-desktop-and-portable.csproj

```diff
@@ -19,6 +19,10 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="NewtonSoft.Json" Version="9.0.1" />
```

Comment:
Please add a comment pointing to the issue that tracks whatever is happening here. @livarcocc mentioned that there is a bug that is fixed by this, but let's make sure it's tracked.